prime-router/src/main/kotlin/cli/SettingCommands.kt

```diff
@@ -553,7 +569,10 @@ class GetMultipleSettings : SettingCommand(
     fun getAll(environment: Environment, accessToken: String): String {
         // get orgs
         val orgsJson = getMany(environment, accessToken, SettingType.ORG, settingName = "")
-        val orgs = jsonMapper.readValue(orgsJson, Array<OrganizationAPI>::class.java)
+        var orgs = jsonMapper.readValue(orgsJson, Array<OrganizationAPI>::class.java)
+        if (filter != null) {
+            orgs = orgs.filter { it.name.startsWith(filter!!, ignoreCase = true) }.toTypedArray()
+        }
 
```

Comment:
Petty simple to do.

Reply:
Yes, this should meet my primary use case, which is to grab all the receivers and senders belonging to one organization.